editor/test/util/eduction_test.clj

```diff
@@ -22,119 +22,194 @@
 (defn- eduction? [value]
   (instance? Eduction value))
 
+;; Note: Equality checks are performed twice to root out issues from repeat
+;; evaluation of the eduction.
+
```

Comment:
Unrelated fix: I discovered this quirk and rewrote all the eduction tests to cover it.